flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricNames.java

```diff
@@ -51,6 +51,8 @@ private MetricNames() {
 	public static final String TASK_SLOTS_TOTAL = "taskSlotsTotal";
 	public static final String NUM_REGISTERED_TASK_MANAGERS = "numRegisteredTaskManagers";
 
+	public static final String NUMBER_OF_RESTARTS = "numberOfRestarts";
```

Comment:
How about `NUM_RESTARTS = "numRestarts"` to be consistent with the other metrics?

Reply:
This comment should be on the other PR.

Reply:
Actually we are not consistent already, e.g., `numberOfFailedCheckpoints`. I think it's ok as it is.